docs/source/install.rst

```diff
@@ -13,51 +13,53 @@ Installation
 Installing with pip
 -------------------
 
-It's possible to install ``libsemigroups_pybind11`` using ``pip`` via one of:
+It's possible to install ``libsemigroups_pybind11`` using ``pip`` something
+like the following (depending on your system and setup):
 
 ::
 
     pip install libsemigroups_pybind11
     pip3 install libsemigroups_pybind11
     python -m pip install libsemigroups_pybind11
```

Comment:
I think this reads as thought you need to run all three lines. Perhaps three separate code blocks would be better?

Reply:
Yeah that seems sensible.